source/common/http/conn_pool_grid.h

```diff
@@ -95,6 +92,12 @@ class ConnectivityGrid : public ConnectionPool::Instance,
                                   absl::optional<Http::Protocol> protocol);
 
   private:
+    // Removes this from the owning list, deleting it.
+    void deleteThis();
+
+    // Marks HTTP/3 broken if the HTTP/3 attempt failed but a TCP attempt succeeded.
```

Comment:
nit: do we define somewhere what "broken" means?

Reply:
Added a comment which explains the impact of HTTP/3 being broken which hopefully helps.

Reply:
It does, thanks!